File: Soal-5/PRAK105-2410817310014-INDRA_SURYADILAGA.c
#include <stdio.h>

int main() {
    int a = 5, b = 6, x = 7, y = 8;
    
    printf("Variabel a bernilai %d\n",a);
    printf("Variabel b bernilai %d\n",b);
    printf("Variabel x bernilai %d\n",x);
    printf("Variabel y bernilai %d\n",y);
    printf("Total sisa bagi dari a dibagi b dan x dibagi y adalah %d", (a%b)+(x%y));
    return 0;
}

File: Soal-5/PRAK105-2410817310014-INDRA_SURYADILAGA.py
a=5
b=6
x=7
y=8

charvar = ['a', 'b', 'x', 'y']
datavar = [a, b, x, y]

for char, data in zip(charvar, datavar):
    print(f'Variabel {char} bernilai {data}')

print(f'Total sisa bagi dari a dibagi b dan x dibagi y adalah {(a%b)+(x%y)}')
